content/04.study.md

```diff
@@ -47,6 +47,26 @@ Deep learning applied to gene expression data is still in its infancy, but the f
 
+### DNA Methylation (to remove after merge)
+
+####  Latent Space Construction
+
+In addition to the prediction of methylation sites and levels, and classification and regression of these profiles, new work has extracted biologically significant features from methylation data using unsupervised deep learning. 
+Particularly, unsupervised methods focus on construction of a latent space that semantically encodes biologically important features from methylation profiles. 
+One method used several stacked binary restricted Boltzmann machines (forming a deep neural network) to learn a low-dimensional subspace representation of the methylation profiles of 5000 CpG sites with highest variance across 136 women breast tissue samples, 113 breast cancer samples and 23 non-cancerous samples, and samples in the latent space were clustered (via self-organizing maps) to show that the latent space could differentiate breast cancer samples from non-neoplastic samples. 
+Furthermore, the latent space was visualized using t-SNE (t-distributed stochastic neighbor embedding) [@arxiv:1808.01359]. 
+The other method for latent space construction was inspired by Way and Greene’s work on Variational Auto-Encoders in the ovarian and pancancer setting for RNASeq data, which in addition to learning a latent semantic space, used vector subtraction to identify biologically relevant features that stratify patient sex, metastatic activation, cancer types and tumor subtypes, amongst others. 
+
+Titus et. al. adopted the VAE developed by Way et. al. and modified it to perform dimensionality reduction on 300,000 PAM50-assigned CpG features to 100 latent features in 862 samples, performing t-SNE visualizations, clustering, and classifying tumor subtypes from a Breast Cancer related TCGA dataset. 
+Titus extended this work to find the 100-dimensional latent space of 100k CpG sites across around 1200 samples, and selected latent space dimensions that were the most highly associated with the differentiation between estrogen-response (ER) positive and negative tumor samples in breast cancer patients to see if the latent space could serve as a useful tool for indicating possible responses to endocrine therapy. 
+After backtracking these VAE dimensions to their highly contributing CpG loci, the pathways that were found at most of these loci were significantly associated with ER-status, the VAE dimensions were associated with CpGs of sparse non-coding regions, and a nonredundant set of CpGs were identified as being related to ER-positive and ER-negative status, suggesting VAE’s importance in epigenetic work for diagnosis, risk, prognosis, and treatment response ascertainment [@doi:10.5220/0006636401400145; @doi:10.1101/433763]. 
+Another study explored the latent features of lung cancer methylation profiles that were extracted using variational autoencoders, and were able to classify cancer subtypes [@doi:10.1109/BIBM.2018.8621365].
```

Comment:
It would also be nice to have a quick summary in this paragraph of the strengths and weaknesses of the various types of approaches if there's enough literature out there to identify them.

Reply:
I've added a brief summary, but not specifically addressing the strengths and weaknesses at this time because there is not yet an abundance (although growing quickly) of literature in this space yet. It is an emerging application area.